files/image_config/sysctl/sysctl-net.conf

```diff
@@ -32,6 +32,10 @@ net.ipv6.conf.eth0.accept_dad=0
 net.ipv6.conf.default.keep_addr_on_down=1
 net.ipv6.conf.all.keep_addr_on_down=1
 net.ipv6.conf.eth0.keep_addr_on_down=1
+net.ipv6.conf.default.disable_ipv6=1
```

Comment:
Based on the review comment, have we agreed to enable IPv6 by default and disable based on some profile?

https://github.com/Azure/SONiC/pull/625#discussion_r433997492

Reply:
As per design change, enabled the ipv6 by default.